pkg/virt-config/config-map.go

```diff
@@ -60,7 +60,7 @@ const (
 	MigrationCompletionTimeoutPerGiB         int64  = 800
 	DefaultMachineType                              = "q35"
 	DefaultCPURequest                               = "100m"
-	DefaultMemoryRequest                            = "8Mi"
```

Comment:
We were talking about a default value of 1Gi too. Is that off the table?

Reply:
I think so, at least at the level of kubevirt-core.

When one of guest-memory or hugepages is provided, the 'cluster memory-overcommit' ratio would be applied to default memory-request so there will be no use of such default setting.

When neither guest-memory nor hugepages nor memory-request is provided, I think it makes more sense to fail the request rather than default it to a pre-define value as it really depends on what is inside the guest, assuming clients would most probably default it to a reasonable value appropriate for their users or use common-templates that should default it according to the operating system being used.